api/src/main/java/io/serverlessworkflow/api/serializers/WorkflowSerializer.java

```diff
@@ -159,8 +159,12 @@ public void serialize(Workflow workflow, JsonGenerator gen, SerializerProvider p
       gen.writeEndArray();
     }
 
-    if (workflow.getConstants() != null && !workflow.getConstants().getConstantsDef().isEmpty()) {
-      gen.writeObjectField("constants", workflow.getConstants().getConstantsDef());
+    if (workflow.getConstants() != null) {
```

Comment:
Is it possible to write test for this in WorkflowToMarkupTest?

Reply:
sure, done